### DISCUSSION TITLE
Organ property usage in Settings

### DISCUSSION THREAD

**Discussion by paolo2504** [GrandOrgue/grandorgue#1054] [Developers]
Please i ask somebody to explain to me the advantage of having property at Organ level (accessible in Setting/Organs...Property).
I mean: access Midi property or initial Midi property does make sense of course. But what is the utility of setting property at organ level ? My apologies, i don't understand.

![immagine](https://user-images.githubusercontent.com/50132661/155584415-03ef0f5d-1879-4cc6-9eea-39f7261ed39e.png)

**Comment by willeke1234** [comment]
The Help says:

> Properties
This button opens the Midi Event Editor to manage the MIDI event(s) which enable GrandOrgue to load a sampleset when received. 
A use case of this feature can be: use a rotary selector that sends MIDI Program Change messages to easily switch between samplesets.

**Reply by paolo2504** [reply]
Thanks . However I still don’t se the advantage of having property at organ level. Why ? It’s not a keyboard or pedal or stop or anything else: the organ Itself is not a midi device to be configured.

**Reply by willeke1234** [reply]
If a stop has a midi event then the stop can be triggered by sending the midi event. An organ can also be triggered by a midi event. When GrandOrgue receives this midi event, it will load the organ. It's remote control.

Try it: set an event and send the event.

**Reply by paolo2504** [reply]
Ok tks. Now I begin to understand but still don’t see the usefulness. 
regards.
